ios/brave-ios/Sources/Brave/Assets/Images.xcassets/NTT/ntt_pause_button.imageset/Contents.json

```diff
@@ -0,0 +1,22 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "filename" : "Vector@2x.png",
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "filename" : "Vector@3x.png",
+      "idiom" : "universal",
+      "scale" : "3x"
```

Comment:
nit: can we use filenames that match the button type

Reply:
Replaced by Leo SF Symbol icons